### PR TITLE
Add sql vs nosql blog post

### PR DESCRIPTION
**Pull request by ebenezerdon**
## What does this PR do?

- Adds SQL vs NoSQL blog post and assets

## Test Plan

- Run the app and navigate to `/blog/post/sql-vs-nosql`
- Confirm that article and assets are displayed correctly

### CODE REVIEW

File: src/routes/blog/post/sql-vs-nosql/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+
+If you've been wondering whether to use SQL or NoSQL for your next project, you found the right article.
+
+Choosing which database to use is a key part of system design, as each type offers specific advantages and limitations.
+This choice impacts everything from application performance and scalability to data management and operational costs. In this guide, we'll cover SQL and NoSQL databases in detail, examining data structure, scalability, performance, querying capabilities, and practical use cases. By the end, you should have a clearer understanding of which database type best fits your project.
+
+# **Understanding SQL databases**
+
+**SQL databases**, also known as **relational databases**, structure data into tables with rows and columns. Common SQL databases include **MySQL**, **PostgreSQL**, **SQL Server**, and **Oracle**. The relational model links data across tables using **primary** and **foreign keys**, providing a highly organized, consistent data structure.
```
Comment (fogelito) [inline src/routes/blog/post/sql-vs-nosql/+page.markdoc:21]: MariaDB we use for self-hosting

**Review by fogelito — APPROVED** [review]
Niiice 
I think you can point out the differences b when adding/ altering attributes / indexes 
Locks, speed Can be a big PRO & Con WDUT?

File: src/routes/blog/post/sql-vs-nosql/+page.markdoc (hunk 1 above)
Comment (ebenezerdon) [reply src/routes/blog/post/sql-vs-nosql/+page.markdoc]: Added, thanks

**Comment by ebenezerdon** [comment]
> Niiice I think you can point out the differences when adding/altering attributes / indexes Locks, speed Can be a big PRO & Con WDUT?

Thanks @fogelito! I've updated the article with your suggestions

**Review by ernstmul — APPROVED** [review]

**Pull request merged**